datalad/metadata/metadata.py

```diff
@@ -1001,7 +1001,7 @@ def __call__(
     @staticmethod
     def custom_result_renderer(res, **kwargs):
         if res['status'] != 'ok' or not res.get('action', None) == 'metadata':
-            # logging complained about this already
+            generic_result_renderer(res)
```

Comment:
I think this file yet needs an import of the ``generic_result_renderer``

Reply:
Thanks, done!